tests/.prettierignore

```diff
@@ -0,0 +1,6 @@
+# Add files here to ignore them from prettier formatting
+
+yarn.lock
+.next
+
+backend/forms/*/form.*.sef.json
```

Comment:
I guess this isn't needed ?

Reply:
Sorry my mistake, I was copying prittier from `/next` project so now we should have it unified across the project.